### DISCUSSION TITLE
Possibly incorrect accessibility label for reaction buttons

### DISCUSSION THREAD

**Discussion by yncat** [community/community#17158] [Accessibility]
Hello.

In [this discussion thread](https://github.com/facebook/create-react-app/discussions/12424), my first post has a reaction from someone. Screen reader says:

> Upvote: (1): yncat, 01:22PM yesterday

but the reaction isn't mine. I haven't dug into the detailed DOM attributes, but I think the label is something wrong.

The same issue sometimes happens for PR review comments, possibly issue comments, too.

Browser: Chrome, latest
Screen reader: NVDA, latest

Btw, I'm very glad to see the accessibility feedback topic pinned to the entire discussion page. Thanks very much for that!

**Comment by benjiallen** [comment]
Hi @yncat thanks for the feedback. I think you're right, the label for this control is not intuitive. It seems that every time you post, you automatically upvote your own post so that's why you're hearing that label within the example you posted. If you didn't want to upvote your own content, you can deselect the upvote. After deselecting, you'll still hear a slightly weird label but it will not communicate "selected" within the button state.

There are other problems I can hear too. I'll take this feedback back to the team to see if we can improve this labelling.

Thanks again!

**Reply by yncat** [reply]
Hello @benjiallen !
Thanks for your clear answer! The upvote thing is now resolved. I did not know that it was automatically upvoted.

I was able to reproduce another labelling issue.

https://github.com/yncat/github-a11y-experiment/pull/1#pullrequestreview-1007725059

I have created this for testing purposes.

For the PR comment where I approved the changes, the PR sender is reacting with thumbs up. The label looks like:

thumbs up (1): yncat, 12:35AM today
But I think it should be
thumbs up (1): yamahubuki, 12:35AM today

Because the person who reacted is yamahubuki.
There is a text "yamahubuki reacted with thumbs up" in the next line, but the button label above still looks weird to me.
Can you report this to the developers as well?

**Comment by cehfisher** [comment]
Thank you @yncat for your feedback on the labels. [Per your earlier comment](https://github.com/orgs/community/discussions/17158#discussioncomment-2957184), it seems as though this issue has been resolved and can be closed (cc @queenofcorgis). Thanks!
